response_operations_ui/templates/respondent-search/_table.html

```diff
@@ -0,0 +1,24 @@
+<table>
+    <thead>
+        <tr>
+            <th class="table--header--cell">Name</th>
+            <th class="table--header--cell">Email Address</th>
+            <th class="table--header--cell">Status</th>
+        </tr>
+    </thead>
+    <tbody>
+        {% for respondent in respondents %}
+        <tr>
+            <td>
+                <a href="{{ respondent.href }}">{{ respondent.name }}</a>
+            </td>
+            <td>
+                {{ respondent.email }}
+            </td>
+            <td>
+                <span class="status status--small {{ respondent.status_class }}">{{ respondent.status | title }}</span>
```

Comment:
This is one for someone else,  is it correct to have display/css knowledge in the code?  My gut says no, but I don't know what would be better.  Maybe a partial?  That seems like a lot to just cleanly sort out the classes on a tag.

Reply:
Myself, I would say this is right.  The class determination logic isn't in the code, so I have separated concerns.

Reply:
That was my point, I was unsure whether the code should know about styles?  My gut says style information should live within the template, but the way you've done it looks so reasonable, which is why I asked the question.

Reply:
I see what you mean there - my take on it is that I prefer the logic in the view rather than the template itself.

I suppose there could be a middle ground of the class translator in a Jinja2 macro, but unless you feel strongly about it, I'd rather not.